src/main/java/Task.java

```diff
@@ -0,0 +1,22 @@
+public class Task {
+
+    protected String name;
+    protected boolean isDone;
+
+    public Task(String name) {
+        this.name = name;
+        isDone = false;
+    }
+
+    public void changeStatus(boolean isDone) {
```

Comment:
I like the consistent naming of methods using camelCase throughout the code!! :)